Aztec/Classes/NSAttributedString/HTML/HTMLAttributeRepresentation.swift

```diff
@@ -0,0 +1,30 @@
+
+class HTMLAttributeRepresentation: HTMLRepresentation {
+
+    typealias Attribute = Libxml2.Attribute
+    typealias StringAttribute = Libxml2.StringAttribute
+
+    /// The element that owns this attribute.
+    ///
+    var element: HTMLElementRepresentation?
+
+    /// The attribute name.
+    ///
+    let name: String
+
+    /// The attribute's value, if present.
+    ///
+    let value: String?
+
+    init(for attribute: Attribute, in element: HTMLElementRepresentation? = nil) {
+
+        self.element = element
+        name = attribute.name
+
+        if let stringAttribute = attribute as? StringAttribute {
+            value = stringAttribute.value
+        } else {
+            value = nil
+        }
```

Comment:
I can't really use the second mechanism as the initializer needs to assign a value to `value` in all cases (the code can't just `return`).  And even if I set a default value there, adding a guard to an `init` method means you can't initialize anything else below it, if the need arises - it's a bit restrictive.

The first snippet is nicer so I'm going to switch to it.